testui/support/appium_driver.py

```diff
@@ -330,8 +330,10 @@ def start_selenium_driver(
                 driver = webdriver.Remote(command_executor=url, options=options)
             else:
                 if browser.lower() == "chrome":
+                    for key, value in desired_caps.items():
+                        options.set_capability(key, value)
                     driver = webdriver.Chrome(
-                        desired_capabilities=desired_caps, options=options
+                        options=options
                     )
```

Comment:
could be in one line

Reply:
Right, I'll fix that

About 

> I don't see the version update 😅

That should be fixed on a different issue request, about adding requirements.txt to the project, this merge is to fix deprecations due to the selenium and appium update from 3.141.0 to 4.1.3, and 1.2.0 to 2.2.0

Reply:
ah, i assumed that the repo has such file :D definitely need to add it then